app/controllers/hub/unattended_clients_controller.rb

```diff
@@ -1,5 +1,6 @@
 module Hub
   class UnattendedClientsController < ApplicationController
+    FILTER_COOKIE_NAME = "sla_violations_filters".freeze
```

Comment:
One option is to compute this from the class or action name, like 
```ruby
def filter_cookie_name
  self.class.name + "_filters"
end
```

I like the possibility of computing the cookie name without adding a new public attribute to the class itself (`FILTER_COOKIE_NAME`).

If we take this approach, we can define `filter_cookie_name` in `ClientSortable`, which is nice because that's where it's consumed.

I have a general rule that I like code where the number of lines of code between a variable's definition and its use are as low as possible. The best way to win at that metric is to not to define variables, which sometimes is crazy, but in this case, seems good.

Not a blocker.